dev-env.config

```
@@ -1,3 +1,6 @@
 JAVA_VERSION=17
 NODE_VERSION=20.16.0
-POSTGRES_VERSION=12.20
\ No newline at end of file
+POSTGRES_VERSION=12.20
+DB_URL=jdbc:postgresql://localhost:5432/cdd?user=cdd&password=cdd
+DB_USERNAME=cdd
+DB_PASSWORD=cdd
```

[Para arrumar] Seria bom botar `DB_NAME=cdd` aqui também, remove a necessidade de um .env quando o container de desenvolvimento está completamente configurado.

```suggestion
DB_URL=jdbc:postgresql://localhost:5432/cdd?user=cdd&password=cdd
DB_NAME=cdd
DB_USERNAME=cdd
DB_PASSWORD=cdd
```

---

Resolvido no [3c9bab2](https://github.com/c137santos/FATEC-API-3-SEMESTRE/pull/22/commits/3c9bab2934d7f9f6c2857f1eef7f7b94e73926a6)